<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameElementsGenerator</class>
 <widget class="QDialog" name="GameElementsGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Display error message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="displayAnswerVerdictSingleChoiceQuestionLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Aici va fi generata intrebarea!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="multipleChoiceAnswer4Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ans4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputAnswerLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="displayAnswerVerdictMultipleChoiceQuestionLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>corect/incorect</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayProximityCorrectAnswerLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="submitAnswerButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit
Answer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="saveQuestionsInFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save questions in file</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkAnswerSelection">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="chooseTerritoryLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>chooseTerritory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multipleChoiceAnswer2Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ans2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multipleChoiceAnswer1Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ans1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multipleChoiceAnswer3Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ans3</string>
   </property>
  </widget>
  <widget class="QLabel" name="elapsedTimeLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>elapsedTimeLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getRandomQuestionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Random Question</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fifty_fiftyAdvantageButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Use 50-50 advantage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suggestAnswerAdvantageButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Use suggest answer advantage</string>
   </property>
  </widget>
  <widget class="QLabel" name="suggestAnswerAdvantageLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>378</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>suggestAnswerLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offerAnswersAdvantageButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Offer answers advantage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offeredAnswer1Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>of ans1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offeredAnswer2Button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>of ans2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offeredAnswer3Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>of ans3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offeredAnswer4Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>of ans4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
